ingestion-edge/docker/pytest-upload-coverage

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+# Wrapper for pytest that also submits coverage data.
+
+set -eox pipefail
+
+pytest $@
```

Comment:
```suggestion
pytest "$@"
```

Reply:
this will make bash expansion preserve argument groups, in case they contain spaces.